cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -365,6 +375,42 @@ func (s *Supervisor) Start() error {
 	return nil
 }
 
+func (s *Supervisor) getFeatureGates() error {
+	cmd, err := commander.NewCommander(
+		s.telemetrySettings.Logger,
+		s.config.Storage.Directory,
+		s.config.Agent,
+		"featuregate",
+	)
+	if err != nil {
+		return err
+	}
+
+	stdout, stderr, err := cmd.StartOneShot()
+	fmt.Println(stdout, stderr)
```

Comment:
is this instruction needed? Can be removed in a future PR

Reply:
Thanks for catching this, my removal was left in my staging area and wasn't committed. Removed.